server/analytics/base_report/__init__.py

```diff
@@ -389,6 +389,12 @@ def _es_filter_states(self, query, states, must, params):
             'terms': {'state': sorted(states)}
         })
 
+    def _es_filter_content_types(self, query, content_types, must, params):
+        query[must].append({
```

Comment:
Can you add a simple query generation test for this here https://github.com/superdesk/superdesk-analytics/blob/develop/server/analytics/base_report/base_report_test.py#L496